With v8.0.0 geo-tz/all and geo-tz/now do not work

I'm unable to get either `geo-tz/all` or `geo-tz/now` to work in my import statements. Just `geo-tz` works fine.

When I change my import statement to the following with v8.0.0:

```
import { find } from 'geo-tz/all';
```

I get

```
src/app.service.ts:2:22 - error TS2307: Cannot find module 'geo-tz/all' or its corresponding type declarations.

2 import { find } from 'geo-tz/all';
                       ~~~~~~~~~~~~

[5:52:36 PM] Found 1 error. Watching for file changes.
```

Removing the `/all` works fine. `/now` throws the same error.

Definitely running v8.0.0 as I can see the find-all and find-now .js and .ts files in the dist folder in node_modules.

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ This library does an exact geographic lookup which has tradeoffs. The results ar
 
 ## Entry Points
 
-As of version 8, node-geo-tz offers the ability to choose from the three different timezone boundary products that the [timezone-boudary-builder](https://github.com/evansiroky/timezone-boundary-builder) project produces.
+As of version 8, node-geo-tz offers the ability to choose from the three different timezone boundary products that the [timezone-boudary-builder](https://github.com/evansiroky/timezone-boundary-builder) project produces. If using TypeScript, it may be necessary to use the `dist` entry points (see [issue #165](https://github.com/evansiroky/node-geo-tz/issues/165)).
 
 ### Alike Since 1970 (default)
 
@@ -32,6 +32,7 @@ The default data product used by this library are unioned timezones that are ali
 
 ```js
 const { find } = require('geo-tz')
+// const { find } = require('geo-tz/dist/find-1970') // TypeScript-compatible import
 
 find(12.826174, 45.036933)  // ['Asia/Riyadh']
 ```
@@ -44,6 +45,7 @@ A comprehensive dataset is available to query for all timezone identifiers avail
 
 ```js
 const { find } = require('geo-tz/all')
+// const { find } = require('geo-tz/dist/find-all') // TypeScript-compatible import
 
 find(12.826174, 45.036933)  // ['Asia/Aden']
 ```
@@ -56,6 +58,7 @@ A dataset containing a unioned set of timezones that share the same timekeeping 
 
 ```js
 const { find } = require('geo-tz/now')
+// const { find } = require('geo-tz/dist/find-now') // TypeScript-compatible import
 
 find(12.826174, 45.036933)  // ['Europe/Moscow']
 ```

--- a/package.json
+++ b/package.json
@@ -40,6 +40,24 @@
       "import": "./dist/find-now.js",
       "require": "./dist/find-now.js",
       "types": "./dist/find-now.d.ts"
+    },
+    "./dist/find-1970": {
+      "main": "./dist/find-1970.js",
+      "import": "./dist/find-1970.js",
+      "require": "./dist/find-1970.js",
+      "types": "./dist/find-1970.d.ts"
+    },
+    "./dist/find-all": {
+      "main": "./dist/find-all.js",
+      "import": "./dist/find-all.js",
+      "require": "./dist/find-all.js",
+      "types": "./dist/find-all.d.ts"
+    },
+    "./dist/find-now": {
+      "main": "./dist/find-now.js",
+      "import": "./dist/find-now.js",
+      "require": "./dist/find-now.js",
+      "types": "./dist/find-now.d.ts"
     }
   },
   "scripts": {

